middle_end/flambda2.0/to_cmm/un_cps.ml

```diff
@@ -40,6 +40,20 @@ module C = struct
   include Un_cps_helper
 end
 
+(* Type for the result of translating a set of closures *)
+type closure_result = {
+  cmm : Cmm.expression;
+  (* The code whose return value is the set of closure. *)
```

Comment:
Code is a bit too ambiguous in the context of sets of closures. Could you use `expr` as the field label and in the comment ?

Reply:
Renamed to `cmm` (before I saw your comment, ^^)